tests/test.py

```diff
@@ -0,0 +1,30 @@
+import json
+from pyshacl import validate
+import re
+import pytest
+from data import properties
+import rdflib
```

Comment:
Follow Python import conventions as outlined in https://peps.python.org/pep-0008/#imports.